src/languageSupport/languages/influxql/connection.ts

```diff
@@ -53,19 +55,18 @@ export class ConnectionManager extends AgnosticConnectionManager {
       groupedTagValues(this._session.tagValues)
     )
       .map(([key, values]) =>
-        values.map((value: string) => `"${key}" = '${value}'`).join(' AND ')
+        values.map((value: string) => `"${key}" = '${value}'`).join(' OR ')
```

Comment:
This was a bug fix. Changing `AND` to `OR` for same tag key with different tag values.

Reply:
What is this a bug fix for - is there an issue for this?

Reply:
No, I didn't file an issue because I found this bug when I went through this PR. The bug is not exposed to anyone because InfluxQL is behind the feature flag `influxqlUI` and the flag is off currently. 

The bug was, if using `AND` to join tag values with the same tag key, the query will result in zero row. For example,

```sql
SELECT "air_temp_degc"
FROM database_name.retention_policy_name."ndbc"
WHERE time >= now() - 1h
  AND ("station_id" = '13001' AND "station_id" = '13002' AND "station_type" = 'fixed')
```
The above query will result a zero row.

This fix is to change the above query to the following:

```sql
SELECT "air_temp_degc"
FROM database_name.retention_policy_name."ndbc"
WHERE time >= now() - 1h
  AND ("station_id" = '13001' OR "station_id" = '13002' AND "station_type" = 'fixed')
```

Sorry, it is confusing that this bug fix is not related to this PR 😅 I can move this bug fix to a separate PR if it is clearer

Reply:
No, that makes sense! Please keep in the same PR - I don't want to make this more burdensome than it needs to be. Just wanted to make sure there wasn't something else we could close down here.